cristin-project/src/main/java/no/unit/nva/cristin/projects/model/nva/NvaProject.java

```diff
@@ -35,12 +21,8 @@
 
 @SuppressWarnings({"PMD.ExcessivePublicCount","PMD.TooManyFields"})
 @JsonInclude(ALWAYS)
-@JsonPropertyOrder({CONTEXT, ID, TYPE, IDENTIFIERS, TITLE, LANGUAGE, ALTERNATIVE_TITLES, START_DATE, END_DATE,
```

Comment:
I do not see any reason to keep this ordering. I know it was needed for unit tests at some early stage, but now we have improved the tests and order of fields should not be important